### ISSUE TITLE
Chain naming consistency for collectives

### ISSUE THREAD

**Issue by stakeworld** [paritytech/cumulus#2699]
Not  really a bug but more an inconsistency which can be frustrating in automatic scripting: for all chains the used `--chain` name is consistent with naming of the database on disk but not for collectives-polkadot, which uses an _ in the database directory.

For example `--chain polkadot` creates `<basedir>/chains/polkadot`, `--chain bridge-hub-kusama`creates `<basedir>/chain/bridge-hub-kusama` etc.

Only `--chain collectives-polkadot` creates `<basedir>/chains/collectives_polkadot` (with on underscore). I would think it logical to keep a consistent naming scheme?

**Comment by joepetrowski** [comment]
Agreed, change is simple ([here](https://github.com/paritytech/cumulus/blob/211eead5cc3772afac9ab5e55df6df509ce9fcd6/parachains/chain-specs/collectives-polkadot.json#L3)), but obviously breaking for node operators or will require a renaming like [for the Statemint path](https://github.com/paritytech/cumulus/blob/211eead5cc3772afac9ab5e55df6df509ce9fcd6/polkadot-parachain/src/command.rs#L719).
